### ISSUE TITLE
[Crash/optimization] StatusEffect target="NearbyItems" crashes the game (has a workaround)

### ISSUE THREAD

**Issue by Krzeszny** [Regalis11/Barotrauma#6242]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Spawn or activate any item that has a StatusEffect with target="NearbyItems"
Result:
```
Exception: Object reference not set to an instance of an object. (System.NullReferenceException)
Target site: System.Collections.Generic.IEnumerable`1[Barotrauma.ISerializableEntity] GetNearbyTargets(Microsoft.Xna.Framework.Vector2, System.Collections.Generic.List`1[Barotrauma.ISerializableEntity])
```

**Steps To Reproduce**
Spawn this (long delay to make sure it activates only once):
```
  <Item name="Test item" identifier="testitem" category="Equipment" scale="0.5" impactsoundtag="impact_metal_heavy">
	<Upgrade gameversion="0.9.3.0" scale="0.5"/>
	<InventoryIcon texture="Content/Items/InventoryIconAtlas.png" sourcerect="580,770,54,63" origin="0.5,0.5" />
	<Sprite texture="Content/Items/Weapons/weapons_new.png" sourcerect="217,305,40,61" depth="0.55" origin="0.5,0.5" />
	<Body radius="20" height="10" density="30" />
	<LightComponent>
		<StatusEffect type="Always" target="NearbyItems" range="100" stackable="false" delay="5">
			<Remove/>
		</StatusEffect>
	</LightComponent>
  </Item>
```
```
Stack trace: 
   at Barotrauma.StatusEffect.GetNearbyTargets(Vector2 worldPosition, List`1 targets) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\StatusEffects\StatusEffect.cs:line 688
   at Barotrauma.Item.ApplyStatusEffect(StatusEffect effect, ActionType type, Single deltaTime, Character character, Limb limb, Entity useTarget, Boolean isNetworkEvent, Boolean checkCondition, Nullable`1 worldPosition) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Items\Item.cs:line 1455
   at Barotrauma.Item.ApplyStatusEffects(ActionType type, Single deltaTime, Character character, Limb limb, Entity useTarget, Boolean isNetworkEvent, Nullable`1 worldPosition) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Items\Item.cs:line 1419
   at Barotrauma.Item.Update(Single deltaTime, Camera cam) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Items\Item.cs:line 1570
   at Barotrauma.MapEntity.UpdateAll(Single deltaTime, Camera cam) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Map\MapEntity.cs:line 558
   at Barotrauma.GameScreen.Update(Double deltaTime) in <DEV>\Barotrauma\BarotraumaShared\SharedSource\Screens\GameScreen.cs:line 264
   at Barotrauma.GameMain.Update(GameTime gameTime) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GameMain.cs:line 962
   at Microsoft.Xna.Framework.Game.DoUpdate(GameTime gameTime) in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 656
   at Microsoft.Xna.Framework.Game.Tick() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 500
   at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\SDL\SDLGamePlatform.cs:line 92
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior) in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 397
   at Microsoft.Xna.Framework.Game.Run() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 367
   at Barotrauma.Program.Main(String[] args) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\Program.cs:line 58
```

**Version**
v0.14.6.0

**Additional information**
**Workaround**: use `targettags="[tags here]"`
[crashreport (6).log](https://github.com/Regalis11/Barotrauma/files/6759976/crashreport.6.log)

**Comment by itchyOwl** [comment]
Fixed in Regalis11/Barotrauma-development@c25bfc7.

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
